includes/functions/plugins/issue-status.php

```diff
@@ -49,13 +49,23 @@ function register_taxonomy() {
 		'search_items'               => __( 'Search Issue Statuses', 'eight-day-week-print-workflow' ),
 		'all_items'                  => __( 'All Issue Statuses', 'eight-day-week-print-workflow' ),
 		'edit_item'                  => __( 'Edit Issue Status', 'eight-day-week-print-workflow' ),
+		'view_item'                  => __( 'View Issue Status', 'eight-day-week-print-workflow' ),
+		'add_or_remove_items'        => __( 'Add or Remove Issue Status', 'eight-day-week-print-workflow' ),
 		'update_item'                => __( 'Update Issue Status', 'eight-day-week-print-workflow' ),
 		'add_new_item'               => __( 'Add New Issue Status', 'eight-day-week-print-workflow' ),
 		'new_item_name'              => __( 'New Issue Status Name', 'eight-day-week-print-workflow' ),
 		'menu_name'                  => __( 'Issue Status', 'eight-day-week-print-workflow' ),
 		'separate_items_with_commas' => '',
 		'choose_from_most_used'      => __( 'Choose from an existing Issue Status', 'eight-day-week-print-workflow' ),
 		'not_found'                  => __( 'No Issue Statuses found.', 'eight-day-week-print-workflow' ),
+		'name_field_description'	 => __( 'The name is how it appears on your site.', 'eight-day-week-print-workflow' ),
```

Comment:
```suggestion
		'name_field_description'     => __( 'The name is how it appears when editing issues.', 'eight-day-week-print-workflow' ),
```

* Spaces for inline alignment (was a tab which I suspect was a typo :) 
* The status isn't publicly queryable so is only shown within the admin, I've edited to reflect this.